Refresh.GameServer/Endpoints/ApiV3/ResourceApiEndpoints.cs

```diff
@@ -60,19 +60,23 @@ public class ResourceApiEndpoints : EndpointGroup
     public Response DownloadGameAssetAsImage(RequestContext context, IDataStore dataStore, GameDatabaseContext database,
         [DocSummary("The SHA1 hash of the asset")] string hash)
     {
-        if (!CommonPatterns.Sha1Regex().IsMatch(hash)) return HashInvalidError;
-        if (string.IsNullOrWhiteSpace(hash)) return HashMissingErrorResponse;
+        bool isPspAsset = hash.StartsWith("psp/");
```

Comment:
This won't ever be hit, since `assets/psp/{hash}/image` can't resolve to this endpoint. You can add a new `ApiV3Endpoint` to resolve that problem.

Reply:
My solution was just to URL encode it as `psp%2Fhash`, thats how i was testing, but yeah i can add a new endpoint, since that would be much cleaner